src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java

```diff
@@ -125,6 +125,12 @@
     public final Histogram bytesValidated;
     /** histogram over the number of partitions we have validated */
     public final Histogram partitionsValidated;
+    /** Coordinator read metrics */
+    public final Timer coordinatorReadLatency;
+    /** Coordinator range metrics */
+    public final Timer coordinatorScanLatency;
+    /** Coordinator write metrics */
+    public final Timer coordinatorWriteLatency;
 
```

Comment:
should we include `waitingOnFreeMemtableSpace` as well?

Reply:
good catch! added